.github/workflows/build.yml

```diff
@@ -0,0 +1,44 @@
+# -----------------------------------------------------------------------------
+#   - invoked on push, pull_request, or manual trigger
+#   - test under at least 3 versions of python
+# -----------------------------------------------------------------------------
+name: build
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
Get rid of the older versions.

Reply:
I'll do that if we need to. Currently there's nothing preventing it from working in 3.7